### PR TITLE
Generalize test fixtures.

### PR DESCRIPTION
**Pull request by morgabra**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ config.log
 *.os
 build.py
 build-*.py
+
```
Comment (pquerna) [inline .gitignore]: empty line

File: src/internal/xjwt_keyset.h [hunk 2]
```diff
@@ -25,11 +25,11 @@ extern "C" {
 #endif /* __cplusplus */
 
 struct xjwt_keyset_t {
-  xjwt_key_t* keys;
+  xjwt_key_t *keys;
```
Comment (pquerna) [inline src/internal/xjwt_keyset.h]: did clang format do this?

File: src/internal/xjwt_keyset.h [hunk 3]
```diff
@@ -25,11 +25,11 @@ extern "C" {
...
 };
 
-XJWT_API(xjwt_key_t*)
-xjwt_keyset__get_by_keyid(xjwt_keyset_t* ks, const char* keyid);
+XJWT_API(xjwt_key_t *)
+xjwt_keyset__get_by_keyid(struct xjwt_keyset_t *ks, const char *keyid);
```
Comment (pquerna) [inline src/internal/xjwt_keyset.h]: this should be type'defed so we don't need the struct bit here?

Comment (morgabra) [reply src/internal/xjwt_keyset.h]: I thought so too, but it was complaining about it. Does it work for you on OSX?

File: src/internal/xjwt_keyset.h (hunk 2 above)
Comment (morgabra) [reply src/internal/xjwt_keyset.h]: Yeah, I can make my editor match the format command I think, one sec.

Comment (morgabra) [reply src/internal/xjwt_keyset.h]: Oh:

clang-format figures it out from other files I think, unless we do something like: 

```
DerivePointerAlignment: false
PointerAlignment: Left
```

Do you want to pick one and reformat everything?

Comment (pquerna) [reply src/internal/xjwt_keyset.h]: there's a scons target with the clang-format settings, `scons format`, i guess we should just check-in `.clang-format` file.

File: tests/test_keyset.c [hunk 4]
```diff
@@ -14,8 +14,8 @@
  *  limitations under the License.
  */
 
-#include "xjwt/xjwt.h"
 #include "internal/xjwt_keyset.h"
+#include "xjwt/xjwt.h"
```
Comment (pquerna) [inline tests/test_keyset.c]: undo this re-ordering?

**Review by pquerna — APPROVED** [review]
LGTM

**Pull request merged**